src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
Adding -beta is fine and expected for contracts that are still under development.  You won't be able to mark this stable until X509Certificates is stable, but that's expected to happen at the same time: ASP.NET v5 RTM.  Assuming you don't plan to ship a stable version of this contract before then, go ahead.